### PR TITLE
Cosmetic changes to logo for dark mode

### PR DESCRIPTION
**Pull request by savidini**
The background of the coordinate system in the logo has been removed in order to display correctly in dark mode on github and in the documentation.

Before:
Screenshot 2024-09-27 at 15-26-45 dfki-ric_pytransform3d 3D transformations for Python

After:
Screenshot 2024-09-27 at 15-44-41 savidini_pytransform3d 3D transformations for Python

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="https://raw.githubusercontent.com/dfki-ric/pytransform3d/main/doc/source/_static/logo.png" />
```
Comment (AlexanderFabisch) [inline README.md:1]: This is intentional so that the image is linked correctly in the long description for PyPI. I don't have a better solution so far. Could you revert this change?

**Comment by AlexanderFabisch** [comment]
Awesome @savidini , thank you!

However, could you revert the change to the readme?

**Comment by savidini** [comment]
@AlexanderFabisch No problem, I have reverted the commit. 
Thanks for the work on the project, it's a great help!

**Comment by AlexanderFabisch** [comment]
Thank you :)
